{"name":"Тест: полёт уточки с неопределёнными крыльями и существующим id","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is org.h2.jdbc.JdbcSQLException: Соединение разорвано: \"java.net.ConnectException: Connection refused: getsockopt: localhost:9092\"\nConnection is broken: \"java.net.ConnectException: Connection refused: getsockopt: localhost:9092\" [90067-196]","trace":"com.consol.citrus.exceptions.TestCaseFailedException: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is org.h2.jdbc.JdbcSQLException: Соединение разорвано: \"java.net.ConnectException: Connection refused: getsockopt: localhost:9092\"\nConnection is broken: \"java.net.ConnectException: Connection refused: getsockopt: localhost:9092\" [90067-196]\r\n\tat com.consol.citrus.DefaultTestCase.executeAction(DefaultTestCase.java:143)\r\n\tat com.consol.citrus.DefaultTestCaseRunner.run(DefaultTestCaseRunner.java:125)\r\n\tat com.consol.citrus.TestActionRunner.$(TestActionRunner.java:35)\r\n\tat autotests.clients.DuckActionsClient.databaseUpdate(DuckActionsClient.java:34)\r\n\tat autotests.tests.duckActionController.FlyDuckTest.flyUndefinedDB(FlyDuckTest.java:78)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.springframework.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:282)\r\n\tat com.consol.citrus.testng.TestNGHelper.lambda$invokeTestMethod$1(TestNGHelper.java:82)\r\n\tat com.consol.citrus.common.DefaultTestLoader.lambda$doLoad$1(DefaultTestLoader.java:116)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat com.consol.citrus.common.DefaultTestLoader.doLoad(DefaultTestLoader.java:116)\r\n\tat com.consol.citrus.common.DefaultTestLoader.load(DefaultTestLoader.java:94)\r\n\tat com.consol.citrus.testng.TestNGHelper.invokeTestMethod(TestNGHelper.java:83)\r\n\tat com.consol.citrus.testng.spring.TestNGCitrusSpringSupport.run(TestNGCitrusSpringSupport.java:185)\r\n\tat com.consol.citrus.testng.spring.TestNGCitrusSpringSupport.run(TestNGCitrusSpringSupport.java:101)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:272)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:669)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: com.consol.citrus.exceptions.CitrusRuntimeException: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is org.h2.jdbc.JdbcSQLException: Соединение разорвано: \"java.net.ConnectException: Connection refused: getsockopt: localhost:9092\"\nConnection is broken: \"java.net.ConnectException: Connection refused: getsockopt: localhost:9092\" [90067-196]\r\n\tat com.consol.citrus.actions.ExecuteSQLAction.executeStatements(ExecuteSQLAction.java:104)\r\n\tat com.consol.citrus.actions.ExecuteSQLAction.doExecute(ExecuteSQLAction.java:73)\r\n\tat com.consol.citrus.actions.AbstractDatabaseConnectingTestAction.execute(AbstractDatabaseConnectingTestAction.java:91)\r\n\tat com.consol.citrus.DefaultTestCase.executeAction(DefaultTestCase.java:136)\r\n\t... 38 more\r\nCaused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is org.h2.jdbc.JdbcSQLException: Соединение разорвано: \"java.net.ConnectException: Connection refused: getsockopt: localhost:9092\"\nConnection is broken: \"java.net.ConnectException: Connection refused: getsockopt: localhost:9092\" [90067-196]\r\n\tat org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:83)\r\n\tat org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:376)\r\n\tat org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:431)\r\n\tat com.consol.citrus.actions.ExecuteSQLAction.executeStatements(ExecuteSQLAction.java:97)\r\n\t... 41 more\r\nCaused by: org.h2.jdbc.JdbcSQLException: Соединение разорвано: \"java.net.ConnectException: Connection refused: getsockopt: localhost:9092\"\nConnection is broken: \"java.net.ConnectException: Connection refused: getsockopt: localhost:9092\" [90067-196]\r\n\tat org.h2.message.DbException.getJdbcSQLException(DbException.java:345)\r\n\tat org.h2.message.DbException.get(DbException.java:168)\r\n\tat org.h2.engine.SessionRemote.connectServer(SessionRemote.java:457)\r\n\tat org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:334)\r\n\tat org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)\r\n\tat org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)\r\n\tat org.h2.Driver.connect(Driver.java:69)\r\n\tat java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)\r\n\tat java.sql/java.sql.DriverManager.getConnection(DriverManager.java:191)\r\n\tat org.springframework.jdbc.datasource.DriverManagerDataSource.getConnectionFromDriverManager(DriverManagerDataSource.java:155)\r\n\tat org.springframework.jdbc.datasource.DriverManagerDataSource.getConnectionFromDriver(DriverManagerDataSource.java:146)\r\n\tat org.springframework.jdbc.datasource.AbstractDriverBasedDataSource.getConnectionFromDriver(AbstractDriverBasedDataSource.java:205)\r\n\tat org.springframework.jdbc.datasource.SingleConnectionDataSource.initConnection(SingleConnectionDataSource.java:224)\r\n\tat org.springframework.jdbc.datasource.SingleConnectionDataSource.getConnection(SingleConnectionDataSource.java:164)\r\n\tat org.springframework.jdbc.datasource.DataSourceUtils.fetchConnection(DataSourceUtils.java:159)\r\n\tat org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:117)\r\n\tat org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:80)\r\n\t... 44 more\r\nCaused by: java.net.ConnectException: Connection refused: getsockopt\r\n\tat java.base/sun.nio.ch.Net.pollConnect(Native Method)\r\n\tat java.base/sun.nio.ch.Net.pollConnectNow(Net.java:682)\r\n\tat java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)\r\n\tat java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592)\r\n\tat java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)\r\n\tat java.base/java.net.Socket.connect(Socket.java:751)\r\n\tat org.h2.util.NetUtils.createSocket(NetUtils.java:103)\r\n\tat org.h2.util.NetUtils.createSocket(NetUtils.java:83)\r\n\tat org.h2.engine.SessionRemote.initTransfer(SessionRemote.java:115)\r\n\tat org.h2.engine.SessionRemote.connectServer(SessionRemote.java:453)\r\n\t... 58 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg0","value":"null"}],"start":1738260366878,"stop":1738260368329,"uuid":"7988ca34-7945-498e-8dec-b6b18a2c2110","historyId":"9832318da973c3d69097b72055018ab5","fullName":"autotests.tests.duckActionController.FlyDuckTest.flyUndefinedDB","labels":[{"name":"package","value":"autotests.tests.duckActionController.FlyDuckTest"},{"name":"testClass","value":"autotests.tests.duckActionController.FlyDuckTest"},{"name":"testMethod","value":"flyUndefinedDB"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"AssemblySystem"},{"name":"subSuite","value":"autotests.tests.duckActionController.FlyDuckTest"},{"name":"host","value":"DESKTOP-RV15V4F"},{"name":"thread","value":"22808@DESKTOP-RV15V4F.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"feature","value":"Endpoint /api/duck/action/fly"},{"name":"epic","value":"Tests for duckActionController"}],"links":[]}